src/main/src/main/java/org/geoserver/system/status/MetricInfo.java

```diff
@@ -18,7 +18,7 @@
     LOGICAL_CPUS("CPU", 101, "Number of logical CPUs"),
     RUNNING_PROCESS("CPU", 102, "Number of running process"),
     RUNNING_THREADS("CPU", 103, "Number of running threads"),
-    CPU_LOAD("CPU", 104, "CPU average load", "%"),
+    // removed from OSHI    CPU_LOAD("CPU", 104, "CPU average load", "%"),
```

Comment:
should this just be removed from the enum?

Reply:
Yep please